python/tk_multi_loader/model_item_data.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk import TankError
+from sgtk.platform.qt import QtCore
+
+shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
+
+
+def get_item_data(item):
+    """
+    Extracts and standardizes the Shotgun data and field value from an item.
+
+    :param item: Selected item or model index.
+    :return: Standardized `(Shotgun data, field value)` extracted from the item data.
```

Comment:
adding some examples here what 'standardized' means would be useful!

maybe those various examples that got deleted in other parts of the code could be shown here?